pkg/kando/location_pull.go

```diff
@@ -79,3 +80,14 @@ func locationPull(ctx context.Context, p *param.Profile, path string, target io.
 func kopiaLocationPull(ctx context.Context, backupID, path string, target io.Writer) error {
 	return kopia.Read(ctx, backupID, path, target)
 }
+
+// connectToKopiaServer connects to the kopia server with given creds
+// nolint:unused,deadcode
+func connectToKopiaServer(ctx context.Context, kp *param.Profile) error {
+	log.Debug().Print("Connecting to kopia server")
```

Comment:
This may cause issues during `locationPull` if we set the log level to debug. Since the data is coming through `stdout` and we are trying to print to `stdout`, the DB data might get affected.
We should remove the log to make it work.

Reply:
Oh ok, will remove that